docs/_config.yml

```
@@ -6,3 +6,5 @@ url: https://UCL-ARC.github.io/python-tooling
 
 aux_links:
   Source repository: https://github.com/UCL-ARC/python-tooling
+
+logo: "https://raw.githubusercontent.com/UCL-ARC/python-tooling/main/images/logo.png"
```

If keeping this, quotes add nothing
```suggestion
logo: https://raw.githubusercontent.com/UCL-ARC/python-tooling/main/images/logo.png
```

---

My understanding was that its [recommended to quote strings including special characters like `:`](https://stackoverflow.com/a/22235064) in YAML files and the `just-the-docs` [documentation example included quotes ](https://just-the-docs.github.io/just-the-docs/docs/configuration/#site-logo) so I followed this.

---

IMO that seems very cautious. I've quite enjoyed removing them, and only using them when needed. So for templating

Needed:
```
variable_a: "{{some_var}}/path/to/folder"
```

Not needed:
```
variable_b: path/to/folder/{{some_var}
```

I find it easier to read without, so only include when actually required. But I'm indifferent as to what you do.